airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -214,7 +191,9 @@ class FullRefreshConfig(BaseConfig):
     configured_catalog_path: Optional[str] = configured_catalog_path
     timeout_seconds: int = timeout_seconds
     deployment_mode: Optional[str] = deployment_mode
-    ignored_fields: Optional[Mapping[str, List[IgnoredFieldsConfiguration]]] = ignored_fields
+    ignored_fields: Optional[Mapping[str, List[IgnoredFieldsConfiguration]]] = Field(
```

Comment:
The "compare records" between "actual" and "expected_records" and between "first read" and "second read" is different hence why we are keeping this. It is worth wondering if those should be aligned but for now, we will assume this is out of scope for this change